src/data/StudentGroup.java

```diff
@@ -6,10 +6,6 @@
 	private List<Student> students;
 	private String name;
 
-	public StudentGroup() {
```

Comment:
Deze lege constructor is nodig voor de import met JSON

Reply:
ah oke